object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -717,7 +718,6 @@ private void checkMustCall(
           .noneMatch(localVarTree -> localVarWithTreeSet.contains(localVarTree))) {
         LocalVarWithTree firstlocalVarWithTree = localVarWithTreeSet.iterator().next();
         reportedMustCallErrors.add(firstlocalVarWithTree);
-
```

Comment:
remove whitespace change?

Reply:
Sorry, didn't notice I had made these (I tried a few different solutions for this before I settled on the one in the PR...).

I kind of prefer the whitespace changes, but I'm happy to revert them if you feel strongly.

Reply:
Keeping is ok with me